defaults/main.yml

```diff
@@ -2,5 +2,5 @@
 # Pin-Priority of NodeSource repository
 nodejs_nodesource_pin_priority: 500
 
-# 0.10 or 0.12 or 4.x
-nodejs_version: "4.2"
+# 0.10 or 0.12 or 4.x or 5.x
+nodejs_version: "5.0"
```

Comment:
Scratch that, we require 5.x for the repo name, and a numeric value 5.0 for the apt task install.